### PR TITLE
Mpd 1352 pull transfers commitments data to rds

### PR DESCRIPTION
**Pull request by kuzinator**

### CODE REVIEW

File: src/SFA.DAS.Data.Functions/ProcessTransferRelationshipStartMessage.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.ServiceBus.Messaging;
+
+namespace SFA.DAS.Data.Functions
+{
+    public class ProcessTransferRelationshipStartMessage
+    {
+        [FunctionName("ProcessTransferRelationshipStartMessage")]
+        public static void Run([ServiceBusTrigger("transfer_connection_invitation_sent", "RDS_TransferConnectionInvitationSentProcessor", AccessRights.Manage,Connection = "MessageBusConnectionString")]string paymentCreatedMessage, ExecutionContext executionContext, TraceWriter log)
```
Comment (MichaelYoung1981) [inline src/SFA.DAS.Data.Functions/ProcessTransferRelationshipStartMessage.cs]: Is this actually doing anything?  I may be being a bit thick!

File: src/SFA.DAS.Data.Domain/Models/ApprenticeshipEvent.cs [hunk 2]
```diff
@@ -1,4 +1,5 @@
 ﻿using System;
+using SFA.DAS.Events.Api.Types;
```
Comment (MichaelYoung1981) [inline src/SFA.DAS.Data.Domain/Models/ApprenticeshipEvent.cs]: We should get rid of this class, it's utterly pointless and inconsistent.  I would have left it but the automapper config changes made me change my mind! :)

File: src/SFA.DAS.Data.Database/Tables/DAS_Employer_Account_Transfers.sql [hunk 3]
```diff
@@ -0,0 +1,13 @@
+﻿CREATE TABLE [Data_Load].[DAS_Employer_Account_Transfers]
```
[the remark this answers is not in the document]
Comment (kuzinator) [reply src/SFA.DAS.Data.Database/Tables/DAS_Employer_Account_Transfers.sql]: Correct me if I'm wrong but account transfers are effectively payments and are collected once a month and don't get updated. Just like payments they don't have IsLatest and UpdateDateTime. Will be benefit from having them there?

File: src/SFA.DAS.Data.Functions/ProcessTransferRelationshipStartMessage.cs (hunk 1 above)
Comment (kuzinator) [reply src/SFA.DAS.Data.Functions/ProcessTransferRelationshipStartMessage.cs]: No, it's not doing anything. It's Steve's legacy and any changes to it here are purely to fix the failing build. There is a proper update and fix coming in another branch that reads transfer relationships from EAS queue

File: src/SFA.DAS.Data.Database/Tables/DAS_Employer_Account_Transfers.sql (hunk 3 above)
[the remark this answers is not in the document]
Comment (MichaelYoung1981) [reply src/SFA.DAS.Data.Database/Tables/DAS_Employer_Account_Transfers.sql]: You're absolutely right.  There is no need for IsLatest but payments have an UpdateDateTime column (I think!) which is useful to see when the data was populated in RDS.

File: src/SFA.DAS.Data.Functions/ProcessTransferRelationshipStartMessage.cs (hunk 1 above)
Comment (MichaelYoung1981) [reply src/SFA.DAS.Data.Functions/ProcessTransferRelationshipStartMessage.cs]: Excellent, thought it would be something like that.

**Comment by ESFA-Pawan** [comment]
This has been already merged into feature branch so closing it - #118

**Pull request closed without merging**